scripts/test-e2e-kubernetes.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# test-e2e-kubernetes.sh
+#
+# SUMMARY
+#
+#   Run E2E tests for Kubernetes.
+
+random-string() {
+  local CHARS="abcdefghijklmnopqrstuvwxyz0123456789"
+  # shellcheck disable=SC2034
+  for i in {1..8}; do
+    echo -n "${CHARS:RANDOM%${#CHARS}:1}"
+  done
+  echo
+}
+
+# Build a docker image if it wasn't provided.
+if [[ -z "${CONTAINER_IMAGE:-}" ]]; then
```

Comment:
Do we not have an existing way to build docker images?

Reply:
We do! This test just invokes, see below.